.github/workflows/tests.yml

```diff
@@ -88,6 +89,9 @@ jobs:
             python-version: '3.13'
             # https://github.com/matplotlib/matplotlib/issues/29732
             pygobject-ver: '<3.52.0'
+          - name-suffix: "(Python 3.14 pre-release)"
```

Comment:
No need for `Python 3.14` in the suffix; it's already there at the start of the name.